nc-games/src/components/Reviews.js

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react"
+import { instance } from "../utils/axios"
+
+export const Reviews = () => {
+    const [reviews, setReviews] = useState([])
+    useEffect(()=>{
+        instance.get('/reviews').then((result)=>{
+            setReviews(result.data.reviews)
+        })
```

Comment:
we could extract this function and put it in an `api.js` and then just call it in the useEffect to keep things tidy and have all your requests to the server in one place - just a suggestion!